### PR TITLE
generate AssetsLockFilePath.flag for traversal projects

### PR DESCRIPTION
**Pull request by kingerja**
#147 

Make traversal import Microsoft.Common.targets
remove after.any.targets import from after.traversal.targets since microsoft.common.targets imports after.any.targets.

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -42,6 +43,7 @@
   <PropertyGroup Condition=" Exists('$(MSBuildProjectDirectory)\version.json') ">
     <EnableNerdbank_GitVersioning Condition=" '$(EnableNerdbank_GitVersioning)' == '' ">true</EnableNerdbank_GitVersioning>
   </PropertyGroup>
+<UsingTask TaskName="Microsoft.Build.Tasks.AssignProjectConfiguration" AssemblyFile="$(MSBuildBinPath)\Microsoft.Build.Tasks.Core.dll" />
```
Comment (jeffkl) [inline Directory.Build.props]: Why is this needed?

File: src/CBT.Traversal/build/Traversal.targets [hunk 2]
```diff
@@ -10,15 +10,28 @@
...
+  <!-- Must import Microsoft.Common.targets for nuget restore of traversal to generate the AssetsLockFilePath.flag. -->
+  <PropertyGroup>
+    <_tmpReBuildDependsOn>$(ReBuildDependsOn)</_tmpReBuildDependsOn>
+    <_tmpBuildDependsOn>$(BuildDependsOn)</_tmpBuildDependsOn>
+    <_tmpCleanDependsOn>$(CleanDependsOn)</_tmpCleanDependsOn>
+  </PropertyGroup>
+
+  <Import Project="$(MSBuildToolsPath)\Microsoft.Common.targets"/>
+
+  <PropertyGroup>
```
Comment (jeffkl) [inline src/CBT.Traversal/build/Traversal.targets]: Why is this needed?

File: Directory.Build.props (hunk 1 above)
Comment (kingerja) [inline Directory.Build.props]: oops I didn't realize I pushed that file.  That wasn't suppose to be included.  Good catch.  Bad on me for not seeing it.

File: src/CBT.Traversal/build/Traversal.targets (hunk 2 above)
Comment (kingerja) [inline src/CBT.Traversal/build/Traversal.targets]: Microsoft.Common.targets chains in the nuget.targets.

**Comment by kingerja** [comment]
rolled into #151
